pkg/tlscheck/tlscheck.go

```diff
@@ -43,11 +43,8 @@ func CheckTLS(ctx context.Context, origAddr string) (bool, error) {
 	conn, err := proxy.AwareDialContextTLS(ctx, fmt.Sprintf("%s:%s", host, port), nil)
 	if err != nil {
 		switch err.(type) {
-		case x509.CertificateInvalidError,
-			x509.HostnameError,
-			x509.UnknownAuthorityError,
-			tls.RecordHeaderError,
-			*tls.CertificateVerificationError:
+		// TODO: revert Go 1.20 change for https://github.com/stackrox/stackrox/commit/5b163bd264197c379acb3b3be63c5e344ab48793#diff-7bc8351b84225aaf8f3f66811f6870eded759036ff293085de708860c89e149a.
+		case x509.CertificateInvalidError, x509.HostnameError, x509.UnknownAuthorityError, tls.RecordHeaderError:
```

Comment:
I suggest to keep the removed item commented out so we don't forget what we lost.

```suggestion
		case x509.CertificateInvalidError,
			x509.HostnameError,
			x509.UnknownAuthorityError,
			tls.RecordHeaderError
			// TODO: uncomment after switching to Go 1.20+ for RHTAP builds.
			//, *tls.CertificateVerificationError
			:
```

Reply:
Now that we use Brew base image for compiling the code, I reverted this change.